log4j-layout-template-json/src/test/java/org/apache/logging/log4j/layout/template/json/JsonTemplateLayoutTimestampTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache license, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the license for the specific language governing permissions and
+ * limitations under the license.
+ */
+package org.apache.logging.log4j.layout.template.json;
+
+import org.apache.logging.log4j.core.LogEvent;
+import org.apache.logging.log4j.core.config.Configuration;
+import org.apache.logging.log4j.core.config.DefaultConfiguration;
+import org.junit.jupiter.api.RepeatedTest;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class JsonTemplateLayoutTimestampTest
```

Comment:
I have two requests here:

1. Move this class to `...log4j.layout.template.json.resolver.TimestampResolverTest` – `JsonTemplateLayoutTest` is pretty bloated and I want to create `*ResolverTest` classes for each individual resolver.
2. Force some concurrent access here by using, e.g., an `ExecutorService`. (`JsonTemplateLayoutConcurrentEncodeTest` might come handy for inspiration.)

Reply:
I moved it and reworked it to target `TimestampResolver` directly.
Regarding a concurrent test; all public methods are marked as synchronized so I don't think that will be necessary. And this issue is present for a single thread as well.

Reply:
Thank you!